### PR TITLE
Add Scroll Sepolia and Mainnet

### PR DESCRIPTION
**Pull request by isabellewei**
- add Scroll network configs
- update metadata with deployed addresses

### CODE REVIEW

File: packages/ethereum-contracts/truffle-config.js [hunk 1]
```diff
@@ -420,6 +435,7 @@ const E = (module.exports = {
         celoscan: process.env.CELOSCAN_API_KEY,
         basescan: process.env.BASESCAN_API_KEY,
         zkevm_polygonscan: process.env.ZKEVM_POLYGONSCAN_API_KEY,
+        scrollscan: process.env.SCROLLSCAN_API_KEY,
```
Comment (d10r) [inline packages/ethereum-contracts/truffle-config.js:438]: Unfortunately this won't work unless we fork https://github.com/rkalis/truffle-plugin-verify and add support (which fortunately wouldn't be difficult though, see e.g. https://github.com/rkalis/truffle-plugin-verify/commit/b2d88609bfde2b35da60a7ac3c123a58fb79ed3c).

**Comment by hellwolf** [comment]
@d10r some conflict to resolve in metadata.

**Comment by d10r** [comment]
this was merged through https://github.com/superfluid-finance/protocol-monorepo/pull/1787, thx @isabellewei !

**Comment by hellwolf** [comment]
@all-contributors please add @isabellewei for code!

**Pull request closed without merging**